### DISCUSSION TITLE
rt command for random tp in other world

### DISCUSSION THREAD

**Discussion by Slodrein** [Zrips/CMI#5456] [Q&A]
I'm a in survival world, i create new portal with command " rt [PlayerName][Farmingworld] " 

But in my console i have error "Cannot find a player with this nickname"

How can use RT between different world ! 

Thank's

**Comment by AkuaFite** [comment]
![image](https://user-images.githubusercontent.com/81054821/115125756-93fd6700-9fca-11eb-9904-571a31c870f0.png)
Let's try that. On my server works perfect.

**Reply by AkuaFite** [reply]
But you need to enable rtp for world you want in config.yml

**Reply by Slodrein** [reply]
does it work between two different worlds for you?

**Reply by AkuaFite** [reply]
Yes, it works.

**Comment by CloudeLecaw** [comment]
It does work allready
If no world in config is defined then the default values for rt are used

You also can define the sizes by command

`setrt (worldName) (center:[x]:[z]) (min:[range]) (max:[range]) (square/circle) (enabled/disabled)`

Example: `cmi setrt farmworld center:100:100 min:200 max:500 circle enabled`
This enables randomteleportation for world farmworld with the centerpoint of teleportation of X100 and Z100 with a minimum range of 200 Block and a maximum range of 500 blocks

Now if your in world world and want to random teleport someone to farmworld simply use `/cmi rt Player farmworld`

Now Player will be teleported to a random Location in farmworld within the defined radius, no matter where he accually stays.

**Comment by mrfloris** [comment]
It isn't [PlayerName], it's [playerName] - You also didn't have a space between the name of the player and the world name.
